src/pineko/parser.py

```diff
@@ -1,10 +1,12 @@
-import yaml
-# ATTENTION: this is a partial copy from 
+# -*- coding: utf-8 -*-
+# ATTENTION: this is a partial copy from
 # https://github.com/NNPDF/nnpdf/blob/ec73c9c5d3765c8b600e3015d3f5d6238dd89400/validphys2/src/validphys/fkparser.py
```

Comment:
Do we actually need this file at the end? Because I'd like to have a single copy, either here or in `vp`

Reply:
Since we will depend on `eko` and `pineappl` I think it is only natural to depend on `pineko`.

I'm fine with adding extra flags that change the behavior as long as the "default" is what vp is doing. Once this PR is merged and we can test that vp is able to simply import this from pineko and that's it we can remove it from vp.

Reply:
On a second thought, this might mean moving `pineko` to `NNPDF` since it will need to be accessible by everyone (as this will be the package that is able to build and read the theories)

Reply:
but the package is already public, so everybody can use it ... and maybe sooner or later we want to publish even on PyPI

Reply:
No strong opinion on this topic, I'm just glad we all agree to remove duplication